tests/camera.nim

```diff
@@ -142,17 +261,17 @@ suite "ImageTracer":
 
     setup:
         var 
-            image: HdrImage = newHdrImage(5, 5)
-            cam: OrthogonalCamera = newOrthogonalCamera(1.2, Transformation.id)
-            im_tr = newImageTracer(image, cam)
+            image = newHdrImage(5, 5)
+            cam = newOrthogonalCamera(1.2, Transformation.id)
+            im_tr = ImageTracer(image: image, camera: cam)
```

Comment:
Ok, getting rid of some useless code, i like it. I'm sorry, but when i define a variable I almost automatically specify its type even though it's not necessary